crates/ln-dlc-node/src/ln/mod.rs

```diff
@@ -30,6 +30,11 @@ pub const JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT_MAX: u64 = 200_000;
 /// See `JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT_MAX` for the maximum channel value.
 pub const LIQUIDITY_MULTIPLIER: u64 = 2;
 
+/// Sats/vbyte rate for position
+///
+/// The coordinator and the app have to align on this to agree on the fees.
+pub const CONTRACT_TX_FEE_RATE: u64 = 4;
```

Comment:
nit: It would be nice if we could configure that value in the `coordinator-settings.toml`

Reply:
(see related comment on `LIQUIDITY_MULTIPLIER`)